app/_config/theme.yml

```
@@ -26,3 +26,13 @@ SilverStripe\LoginForms\EnablerExtension:
     - '$default'
     - 'bambusa'
     - 'starter'
+
+---
+Name: bambusa-simplestyleguide
+After: '#simplestyleguide'
+---
+# Move the style guide on a different URL segment
+SilverStripe\Control\Director:
+  rules:
+    '_styleguide': false
```

```suggestion
    '_styleguide': '->styleguide'
```

It might be be more intuitive if it just redirects you to the new path. Also, setting it to `false` makes it throw an error.

---

Good suggestion ... didn't even know you could do that.